randovania/game_connection/connector/prime_remote_connector.py

```diff
@@ -3,8 +3,9 @@
 import struct
 import uuid
 
+from retro_data_structures.game_check import Game
```

Comment:
for maximum clarity, might it be worth to add `as RDSGame` or something to this import?